ts/todoapp.ts

```diff
@@ -2,14 +2,14 @@ import { storage } from "./storage.js";
 import { setActiveTabListener } from "./tabSwitch.js";
 import { initializeTodos } from "./initializeTodos.js";
 import { displayTodos } from "./displayTodos.js";
-import { demoTodo } from "./demoTodo.js";
+import { demoTodo, TypeTodo } from "./demoTodo.js";
 import { setTodoCreationListener } from "./setTodoCreationListener.js";
 import { setSingleTodoDeletionListener } from "./setSingleTodoDeletionListener.js";
 import { setCompletedTodosDeletionListener } from "./setCompletedTodosDeletionListener.js";
 
 const todos = document.createElement("ul");
 todos.className = "todo-ul";
-let tasks = storage.get("tasks") ?? demoTodo;
+let tasks: string | TypeTodo = storage.get("tasks") ?? demoTodo;
```

Comment:
You perfectly understand that `tasks` in your local storage is JSON array of todos. Meaning that after you deserialize it, it will be array of todos. But here you lie again to yourself and say that it can be a `string` 🤔